lib/ansible/modules/cloud/ovirt/ovirt_job.py

```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016 Red Hat, Inc.
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: ovirt_job
+short_description: Module to manage jobs in oVirt/RHV
+version_added: "2.9"
+author: "Martin Necas (@mnecas)"
+description:
+    - "This module manage jobs in oVirt/RHV. It can also manage steps of the job."
+options:
+    description:
+        description:
+            - "Description of the job."
+        required: true
+    state:
+        description:
+            - "Should the job be present/absent/failed."
+            - "Present have alias started and absent have alias finished, you can use both."
```

Comment:
I would write:  C(started) is alias for C(present). C(finished) is alias for C(absent). Same in the steps.